hw/ip_templates/rv_plic/fpv/tb/coverage_waivers.tcl.tpl

```diff
@@ -61,3 +61,27 @@ assert -name FSMParasiticState::AlertSenderFSMParasiticState_A\
  {!(dut.gen_alert_tx[0].u_prim_alert_sender.state_q inside\
   {Idle, AlertHsPhase1, AlertHsPhase2, PingHsPhase1, PingHsPhase2, Pause0, Pause1}) ->\
   dut.gen_alert_tx[0].u_prim_alert_sender.state_d == Idle}
+
+proc clog2 {NumSrc} {
+  return [expr {ceil(log($NumSrc)/log(2))}]
+}
+
+# These are all the dead nodes in a binary tree. They are dead because the rightmost nodes at the
+# bottom of the tree are tied with 1'b0. Hence, their parents on levels below them also assigned
```

Comment:
I think we have two different orientations here :-) If the leaves are at the bottom of the tree, the parents are above them... (Or put the tree the other way up! I don't think it really matters)